File: src/main/java/org/orgst/Variables/AppData.java
/*
orgST - A fun java console with apps
Copyright (C) 2025  Wdboyes13, MakiDevelops, Table37

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see https://www.gnu.org/licenses/.
 */
package org.orgst.Variables;

import java.util.HashMap;

public class AppData {
    public static HashMap<String, String> AppData;
    static {
        AppData = new HashMap<>();
        AppData.put("Version", "1.8.0");
        AppData.put("Updates", "9");
        AppData.put("Authors", "progman.task, Wdboyes13, chureki");
        AppData.put("Title", "orgST");
        AppData.put("Sponsors", "orgST Public, Archive, CalcTech, toyathing");
        AppData.put("GitHub", "https://github.com/MakiDevelops/orgST");
        AppData.put("Website", "https://orgst.ca");
    }
}

File: orgST++/src/Variables/AppData.cpp
//===------- AppData.cpp - Application Data Code - C++ ------------------------------*-===//
//
// Copyright (C) 2025 Wdboyes13
// Part of Wdboyes13's orgST++, under the GNU General Public License v3.
// See https://weelam.ca/license/ for license information.
// SPDX-License-Identifier: GPL-3.0-or-later
//
//===----------------------------------------------------------------------------------===//
#include <map>

namespace OrgST::Variables {
    std::map<std::string, std::string> AppData;
    void FillAppData(){
        AppData.insert(std::make_pair<std::string, std::string>("Version", "1.8.0"));
        AppData.insert(std::make_pair<std::string, std::string>("Updates", "9"));
        AppData.insert(std::make_pair<std::string, std::string>("Authors", "progman.task, Wdboyes13, chureki"));
        AppData.insert(std::make_pair<std::string, std::string>("Title", "orgST"));
        AppData.insert(std::make_pair<std::string, std::string>("Sponsors", "orgST Public, Archive, CalcTech, toyathing"));
        AppData.insert(std::make_pair<std::string, std::string>("GitHub", "https://github.com/MakiDevelops/orgST"));
        AppData.insert(std::make_pair<std::string, std::string>("Website", "https://orgst.ca"));
    }
}